programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -1324,7 +1324,7 @@ static void run(int argc, char *argv[]) {
 
     // generate tunnel status instance and save active state and start time
     if (config_dir != NULL) {
-        if (!realpath(config_dir, config_dir)) {
+        if (!realpath(config_dir, NULL)) {
```

Comment:
Why not resolve_directory?
And it is a memory leak

Reply:
it's now done in option processing so i removed it and the if below